README.md

```diff
@@ -138,90 +85,71 @@ It takes care of:
   `e2e/project.yaml` under the `setup` key. They are common steps (see their
   description above) and can do things like installing the chart under test,
   setting up required external resources, etc.
+
 * Run tests: invoking the `test` command:
 
 ```
 $ e2e-harness test
 ```
 
-First, the out of cluster tests are executed, if any, defined as
-regular `step` entries under the `outOfClusterTests` key. Then the in cluster tests,
-if they are enabled in `project.yaml`. You should enabled them only if they are
-present, if not enabling them causes an error, see above for a descrition about
-how to write this kind of tests.
+First, the binary and docker image for the project are built. Then the tests
+are compiled, not executed directly. Finally, that binary is executed in the
+test container with an environment defined by the environment variables set
+in the `project.yam` file.
+
 * Teardown: the teardown phase is executing using the `teardown` command.
 
 ```
 $ e2e-harness teardown
 ```
 
-It includes:
-  - Run specific teardown steps: this are defined in the project file
-  `e2e/project.yaml` under the `teardown` key. They are common steps (see their
-  description above) and can do things like removing the chart under test,
-  tearing down required external resources, etc.
-  - Run commomn tear down steps: these differ depending on the mode of operation,
-  for remote, ephemeral clusters they are just deleted, for local clusters tiller
-  and all the required RBAC setup is removed.
+It consists of run commomn tear down steps: these differ depending on the mode
```

Comment:
Better english "It consists of commonly run tear down steps" - I am not exactly sure what you mean here.

Reply:
Yep, finally `consists of running common steps...`